sdks/java/nexmark/src/main/java/org/apache/beam/sdk/nexmark/Main.java

```diff
@@ -145,6 +145,7 @@ public NexmarkPerf decode(InputStream inStream)
         new TableSchema()
             .setFields(
                 ImmutableList.of(
+                    new TableFieldSchema().setName("timestamp").setType("INTEGER"),
```

Comment:
I think we can use the "TIMESTAMP" type here (fits best): https://cloud.google.com/bigquery/data-types

Reply:
+1